Training/Model/Api/UpdateCustomerAttributeCreditHold.php

```diff
@@ -0,0 +1,151 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+declare(strict_types=1);
+
+namespace Codifi\Training\Model\Api;
```

Comment:
**Api** folded is used for interfaces. **Model** folder is used for classes that implemented this interfaces.